src/api/public/apidocs-new/paths/source_project_name_package_name_attribute_attribute_name.yaml

```
@@ -0,0 +1,78 @@
+get:
+  summary: Get package's attribute
+  description: |
+    Get the specified attribute of the package
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - $ref: '../components/parameters/attribute_name.yaml'
+    # Query Strings:
+    # `meta`: either you pass `meta` or not, it's always replaced by meta: 1.
+    - name: rev
+      in: query
+      schema:
+        type: string
+      description: |
+        Passing the number of the revision, this endpoint displays the attributes' XML as it was at that point.
+      example: 3
+    - name: view
+      in: query
+      schema:
+        type: string
+        enum:
+          - blame
+      description: With `view=blame`, information about who changed each XML tag and when is displayed.
+      example: blame
+    - name: with_default
+      in: query
+      schema:
+        type: string
+      description: |
+        If the attribute doesn't contain any value and `with_default` is present,
+        the default values will be displayed, if any.
+      example: 1
+    - name: with_project
+      in: query
+      schema:
+        type: string
+      description: |
+        Passing `with_project`, the response displays the specied attribute of the package's project, if any, in addition to the package one.
+      example: 1
+  responses:
+    '200':
+      description: OK. The request has succeeded.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/source/attributes.yaml'
+          examples:
+            oneAttribute:
+              value:
+                - name: MaintenanceProject
+                  namespace: OBS
+            viewBlame:
+              description: Passing `view=blame`.
+              value: |
+                10 (Admin        2023-03-09 11:46:01     1) <attributes>
+                12 (Iggy         2023-03-13 14:46:01     5)   <attribute namespace="OBS" name="MakeOriginOlder"/>
+                14 (Admin        2023-03-13 15:14:21     6)   <attribute namespace="OBS" name="QualityCategory">
+                14 (Admin        2023-03-13 15:14:21     7)     <value>Development</value>
+                14 (Admin        2023-03-13 15:14:21     8)   </attribute>
+                10 (Admin        2023-03-09 11:46:01     9) </attributes>
+    '400':
+      description: Wrong value for deleted parameter.
+      content:
+        application/xml; charset=utf-8:
+          schema:
+            $ref: '../components/schemas/api_response.yaml'
+          examples:
+            invalid_attribute:
+              value:
+                code: 400
+                summary: "Attribute 'OwnerRootProjectTest' must be in the $NAMESPACE:$NAME style"
```

```suggestion
      description: |
        Bad Request.

        XML Schema used for body validation: [status.xsd](../schema/status.xsd).
      content:
        application/xml; charset=utf-8:
          schema:
            $ref: '../components/schemas/api_response.yaml'
          examples:
            invalid_attribute:
              value:
                code: 400
                summary: "Attribute 'OwnerRootProjectTest' must be in the $NAMESPACE:$NAME style"
              description: Wrong value for deleted parameter.
```
We should add a general description for all 400 errors, and a particular description for each example.

---

Hm... wait. This particular description doesn't fit in that example. :thinking:

---

I realized while addressing what you mentioned before, I updated. Please have another look

---

Fine now. :+1: